frontend/src/components/styles/Card.css

```diff
@@ -0,0 +1,45 @@
+.Card {
+    display: flex;
+    flex-direction: column;
+    background: #eee;
+    width: 300px;
+    padding: 1rem;
+    margin: 1rem;
+    border-radius: 15px;
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19),0 6px 6px rgba(0,0,0,0.23);
+}
+
+.img {
+    object-fit: contain;
+}
+
+.title {
+    font-size: 1.5rem;
+    font-weight: bold;
+    margin: 0.5rem 0;
+}
+
+.description {
+    padding: 0.5rem 0.3rem;
+}
+
+.date {
+    color: grey;
+}
+
+.read_more {
+    margin: 1rem auto 0 auto; 
```

Comment:
Color codes doesn't look good. Either make it bit impressive or use the design provided.

Reply:
@ashishnagpal2498 Now it is fine?
![image](https://user-images.githubusercontent.com/43892590/107032497-eefc2c00-67d9-11eb-8546-496b458d4b32.png)